ansible/roles/ocp4-workload-ocs-operator/tasks/post_workload.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
rather than leaving this empty, can we include the standard debug msg task if you're not doing anything else here?

Reply:
Added the debug task. Sorry, probably got a little over zealous deleting tasks from whatever workload I used as a template.